recipes/extracting-captions-http/src/index.js

```diff
@@ -33,13 +30,11 @@ countertop.addAppliance(VideoHttpIngestionAppliance, {
 })
 countertop.addAppliance(CCExtractorAppliance)
 
-countertop.on(
-	applianceEvents.PAYLOAD,
+countertop.on('data',
 	(payload) => {
 		if (payload.type === dataTypes.TEXT.ATOM) {
 			process.stdout.write(payload.data)
 		}
-	},
-)
+	})
```

Comment:
Could also do something like:
```js
countertop
  .pipe(filterStream(({type}) => type === dataTypes.TEXT.ATOM))
  .pipe(process.stdout);
```

But would need a tidy abstraction around something that reduced to:
```js
function filterStream(fn) {
  return new Transform({
    objectMode: true,
    transform(payload, encoding, callback) {
      if (fn(payload)) {
        this.push(payload);
      }
      callback();
    },
  });
}
```

I usually use external libraries to simplify this boilerplate.